test/e2e/volumes.go

```diff
@@ -170,6 +170,14 @@ func volumeTestCleanup(client *client.Client, config VolumeTestConfig) {
 	}
 
 	if config.serverImage != "" {
+		// Since volume clean up and umount happens in the background.
+		// It is possible that the volume server container is stopped before umount kicks in.
+		// In such case, some volume types (like NFS) is unable to umount the volume.
```

Comment:
we can delete lines 173-175, they are redundant to the info in issue 24100 and the comment on 177....